com.unity.render-pipelines.high-definition/Documentation~/HDRP-Camera.md

```diff
@@ -55,6 +55,7 @@ The HDRP Camera shares many properties with the Unity's [standard Camera](https:
 | **Gate Fit**        | Use the drop-down to select the method that Unity uses to set the size of the resolution gate (aspect ratio of the device you run the application on) relative to the film gate (aspect ratio of the Physical Camera sensor).   **Vertical**: Fits the resolution gate to the height of the film gate. If the sensor aspect ratio is larger than the device aspect ratio, Unity crops the rendered image at the sides. If the sensor aspect ratio is smaller than the device aspect ratio, Unity overscans the rendered image at the sides. If you select this method, changing the sensor width (**Sensor Size** > **X** property) has no effect on the rendered image.<br />&#8226; **Horizontal**: Fits the resolution gate to the width of the film gate. If the sensor aspect ratio is larger than the device aspect ratio, Unity overscans the rendered image on the top and bottom. If the sensor aspect ratio is smaller than the device aspect ratio, Unity crops the rendered image on the top and bottom. If you select this method, changing the sensor height (**Sensor Size** > **Y** property) has no effect on the rendered image.<br />&#8226; **Fill**: Fits the resolution gate to either the width or height of the film gate, whichever is smaller. This crops the rendered image.<br />&#8226; **Overscan**: Fits the resolution gate to either the width or height of the film gate, whichever is larger. This overscans the rendered image.<br />&#8226; **None**: Ignores the resolution gate and uses the film gate only. This stretches the rendered image to fit the device aspect ratio. |
 | **Focal Length**    | Set the distance, in millimeters, between the Camera sensor and the Camera lens. Lower values result in a wider **Field of View**, and vice versa. This property affects [Depth of Field](Post-Processing-Depth-of-Field.md) if you set its **Focus Mode** to **Use Physical Camera**. |
 | **Aperture**        | Use the slider to set the ratio of the f-stop or [f-number](Glossary.md#f-number) aperture. The smaller the value is, the shallower the depth of field is and more light reaches the sensor. This property affects [Depth of Field](Post-Processing-Depth-of-Field.md) if you set its **Focus Mode** to **Use Physical Camera**. This property also affects [Exposure](Override-Exposure.md) if you set its **Mode** to **Use Physical Camera**. |
+| **Focus Distance**     | Set the distance of the focus plane from the Camera. For better compatibiity with older HDRP versions, this property is used in DoF computations only if the **Focus Distance Mode** in the [Depth of Field](Post-Processing-Depth-of-Field.md) volume component is set to **Camera**. |
```

Comment:
Generally, it's best not to mention versions in property docs:

Sets the distance of the focus plane from the Camera. This property is only used in DoF computations if the Focus Distance Mode in the Depth of Field volume component is set to Camera.